### ISSUE TITLE
All post requests into the Database should follow a predefined Schema.

### ISSUE THREAD

**Issue by ethanszeto** [Northeastern-Science-Magazine/nusci-website#17]
* This ticket can only be completed once authentication-JSON is merged into main.

Go through the codebase, and wherever a post request is made to the database, ensure that the request utilizes one of the Schemas predefined in the model folder.
